### PR TITLE
feat: Issue #377 added the ability to accept multiple paths 

### PR DESCRIPTION
**Pull request by aarontravass**
As mentioned in Issue #377, methods can now take an array of paths.

### CODE REVIEW

**Comment by aarontravass** [comment]
Can we have `pnpm format:fix` as part of the pre commit hook?

**Comment by talentlessguy** [comment]
yeah but you should add it in a separate PR

File: .gitpod.yml
Comment (talentlessguy) [inline .gitpod.yml:1]: please delete this file

**Comment by aarontravass** [comment]
> yeah but you should add it in a separate PR

Yes, will do that.

File: packages/router/src/index.ts [hunk 1]
```diff
@@ -206,13 +206,27 @@ export class Router<App extends Router = any, Req = any, Res = any> {
...
     return (...args: RouterMethodParams<Req, Res>): this => {
       const handlers = args.slice(1).flat() as Handler<Req, Res>[]
-      pushMiddleware<Req, Res>(this.middleware)({
-        path: args[0],
-        handler: handlers[0],
-        handlers: handlers.slice(1),
-        method,
-        type: 'route'
-      })
+      if (typeof args[0] == 'object') {
```
Comment (talentlessguy) [inline packages/router/src/index.ts:209]: I would replace this check with `Array.isArray`

File: tests/modules/router.test.ts [hunk 2]
```diff
@@ -373,6 +373,13 @@ describe('Testing HTTP methods', () => {
...
     router.all('/', () => void 0)
 
+    expect(router.middleware[0].type).toBe('route')
+  })
+  it('app.get with array of paths should set GET as HTTP method', () => {
+    const router = new Router()
+
+    router.get(['/', '/test'], () => void 0)
+
     expect(router.middleware[0].type).toBe('route')
```
Comment (talentlessguy) [inline tests/modules/router.test.ts:383]: I think it'd make sense to assert `middleware.length` instead

**Review by talentlessguy — CHANGES_REQUESTED** [review]

File: tests/modules/router.test.ts (hunk 2 above)
Comment (aarontravass) [reply tests/modules/router.test.ts:383]: Is this how the length is checked? 
`expect(router.middleware.length).toHaveLength(2)`

Comment (aarontravass) [reply tests/modules/router.test.ts]: Fixed in commit ad67943ae61c9e428f7bf41ffd424ca2b2d94ddd

**Comment by aarontravass** [comment]
Anything more to be addressed?

**Comment by talentlessguy** [comment]
nope, I'll merge it and release it within next two days

**Pull request merged**